packages/common/src/index.ts

```diff
@@ -1194,6 +1204,24 @@ export const getItemColor = (item: Field | TableCalculation) => {
     return '#0A6640';
 };
 
+export const getResultValues = (
+    rows: ResultRow[],
+    onlyRaw: boolean = false,
+): { [col: string]: any }[] =>
+    rows.map((row: ResultRow) => {
+        const newRow: { [col: string]: any } = {};
+        Object.keys(row).forEach((key: string) => {
+            const value: string = onlyRaw
+                ? row[key]?.value?.raw || row[key]
+                : row[key]?.value?.formatted ||
+                  row[key]?.value?.raw ||
+                  row[key];
```

Comment:
I find this return unexpected. Why do we default to return the entire column object ? 
I was expecting it do be something like:
```
const value: string = onlyRaw
                ? row[key]?.value?.raw 
                : row[key]?.value?.formatted ||
                  row[key]?.value?.raw;
```

Reply:
I also return row[key] in case there is no raw or formatted, as a way of adding some backwards compatibility... but probably not needed